scripts/release_info.py

```diff
@@ -292,24 +303,111 @@ def _check_storage_models(current_release):
+                'PR: for current release' in label_names):
+            return False
+    return True
+
+
+def update_sorted_file(filepath, new_list, start_index_line):
+    """Updates the files AUTHORS and CONTRIBUTORS with a sorted list of
+    new authors or contributors.
+
+    Args:
+        filepath: str. The path of the file to update.
+        new_list: list(str). The list of new authors or contributors to
+            add to the file.
+        start_index_line: str. The line whose index determines the start
+            of authors/contributors list in the file.
+    """
+    content = []
+    with python_utils.open_file(filepath, 'r') as f:
+        content = f.readlines()
+    start_index = content.index(start_index_line) + 2
+    updated_list = new_list + content[start_index:]
+    updated_list = sorted(updated_list, key=lambda s: s.lower())
+    content = content[:start_index] + updated_list
+    with python_utils.open_file(filepath, 'w') as f:
+        for line in content:
+            f.write(line)
+
+
 def main():
     """Collects necessary info and dumps it to disk."""
     branch_name = _get_current_branch()
     if not re.match(r'release-\d+\.\d+\.\d+$', branch_name):
         raise Exception(
             'This script should only be run from the latest release branch.')
 
-    parsed_args = _PARSER.parse_args()
-    if parsed_args.personal_access_token is None:
+    personal_access_token = getpass.getpass(
+        prompt=(
+            'Personal access token for your github ID. '
+            'You can create one at https://github.com/settings/tokens: '))
+
+    if personal_access_token is None:
         python_utils.PRINT(
             'No personal access token provided, please set up a personal '
-            'access token at https://github.com/settings/tokens and pass it '
-            'to the script using --personal_access_token=<token>')
+            'access token at https://github.com/settings/tokens and re-run '
+            'the script')
+        return
+
+    parsed_args = _PARSER.parse_args()
+    if parsed_args.github_username is None:
+        python_utils.PRINT(
+            'No GitHub username provided. Please re-run the '
+            'script specifying a username using --username=<Your username>')
         return
+    github_username = parsed_args.github_username
 
-    personal_access_token = parsed_args.personal_access_token
     g = github.Github(personal_access_token)
     repo = g.get_organization('oppia').get_repo('oppia')
+    repo_fork = g.get_repo('%s/oppia' % github_username)
+
+    blocking_bugs_count = check_blocking_bugs(repo)
+    if blocking_bugs_count:
+        common.open_new_tab_in_browser_if_possible(
+            'https://github.com/oppia/oppia/issues?q=is%3Aopen+'
+            'is%3Aissue+milestone%3A%22Blocking+bugs%22')
+        raise Exception(
+            'There are %s unresolved blocking bugs. Please '
+            'ensure that they are resolved before changelog creation.' % (
```

Comment:
Should this message state that we have opened the browser with a link?

Reply:
We don't do that at other place as well. Should I update the function in `common.py` to always print this message?